embassy-boot/nrf/src/main.rs

```diff
@@ -0,0 +1,46 @@
+#![no_std]
+#![no_main]
+
+use cortex_m_rt::{entry, exception};
+
+#[cfg(feature = "defmt")]
+use defmt_rtt as _;
+
+use embassy_boot_nrf::*;
+use embassy_nrf::nvmc::Nvmc;
+
+#[entry]
+fn main() -> ! {
+    let p = embassy_nrf::init(Default::default());
+    /*
+        for i in 0..10000000 {
+            cortex_m::asm::nop();
+        }
+    */
```

Comment:
I'll add a comment to why it is there: when using debug probe and RTT, this is needed, otherwise writing to flash 'too early' will cause a hard fault.